app/src/main/res/drawable/creator_dashboard_circle.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+  android:shape="oval">
+  <solid
+    android:color="@android:color/white"/>
```

Comment:
Let's keep this transparent in case we evert decide to use a background color other than white.